Google.Api.Generator/RoslynUtils/RoslynConverters.cs

```diff
@@ -93,23 +93,47 @@ private static IEnumerable<ArgumentSyntax> ToArgs(object o)
 
         public static IEnumerable<StatementSyntax> ToStatements(object o)
         {
-            switch (o)
+            if (o is IEnumerable en)
```

Comment:
Possibly move this to a case in the switch, with a possibly-local method for the body?

Reply:
Ah yes, this was the second change to this code. The first change it made more sense moving the `IEnumerable` out of the `switch`. Fixed.